docs/contributing/contributing.md

```diff
@@ -1,111 +1,158 @@
+After all the prerequisites are met, and you are ready to create your pull request, commit and push your work to the branch you have created in your forked repository (if you haven't already done so).  
+Then on the main page of your forked repository, click on the **Compare & pull request** button to open a new Pull Request (on the original Cortex XSOAR Content repository).  
+Fill the description template, and click on **Create pull request**.
 
+For additional information on how to create a Pull Request from a fork, refer to [official GitHub documentation](https://help.github.com/articles/creating-a-pull-request-from-a-fork).
 
-## Open a Pull Request
+### Post-Submission
+After submitting your pull request, sign our [Contributor License Agreement (CLA)](https://github.com/demisto/content/blob/master/docs/cla.pdf), and monitor your pull request.  
 
-After you have completed all the requirements and are ready to open your Pull Request, commit and push your work to the branch you have created in your forked repo. 
+Our Content team will review your code, and might ask questions and request changes.  
 
-Now you can go ahead an open your Pull Request, you can use [this article](https://help.github.com/articles/creating-a-pull-request-from-a-fork/) to do so.
-When creating the pull request make sure to fill in the different section in the pull request template.
+In order to establish a proper release timeframe for your contribution, you are required to respond (and apply the requested changes, if any) within 14 days.  
+Stale pull requests will be closed.
 
-## After opening a Pull Request
+At some point, when the review process is complete (or close to completion), we'll ask to schedule a meeting for an interactive demo.  
 
-After opening the Pull Request, make sure that you:
+Prepare for it, and make sure you have a working instance of Cortex XSOAR, with your contributed pack fully configured.  
+Check out our [Contribution Demo Page](../contributing/demo-prep) for additional information.
 
-- Sign the [CLA](https://github.com/demisto/content/blob/master/docs/cla.pdf): every contributor must sign our Contributor License Agreement in order for their contribution to be added to our content. In case of CLA issues check out our [FAQs](../concepts/faq#cla-is-pending-even-though-i-signed-the-agreement).
-- Monitor your Pull Request on GitHub and be ready for a demo: our Content team will add comments to the Pull Request, asking questions and requesting changes. In order to establish a decent release process for your contribution, you are kindly asked to respond to the reviewer's code review and apply the required changes within 14 days. Stale Pull Requests might be closed.
-- At some point, we'll ask you to schedule a meeting to see an interactive demo. Make sure you have a working installation of Cortex XSOAR with your pack fully configured. Check out our [Contribution Demo Page](../contributing/demo-prep) for more details.
+:::caution
```

Comment:
Caution is not the correct word to use.  It insinuate that something bad will happen.
Use "Note:"  or "Important:"